napalm/base/test/models.py

```diff
@@ -86,6 +86,7 @@
     "remove_private_as": bool,
     "prefix_limit": dict,
     "neighbors": dict,
+    "nhs": bool,
```

Comment:
This change is going to break the tests for all the other drivers - see, for example, https://travis-ci.org/github/napalm-automation/napalm/jobs/679590322#L501